frontend/containers/account-type-selector/component.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC, useMemo } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import noop from 'lodash-es';
+
+import AccountTypeItem, { AccountTypeItemProps } from './account-type-item';
+import { AccountTypeSelectorProps } from './types';
+
+export const AccountTypeSelector: FC<AccountTypeSelectorProps> = ({
+  className,
+  onAccountTypeSelected = noop,
+}: AccountTypeSelectorProps) => {
+  const intl = useIntl();
+
+  const ACCOUNT_TYPES: AccountTypeItemProps[] = useMemo(
+    () => [
+      {
+        accountTypeId: 'investor',
+        name: intl.formatMessage({ defaultMessage: 'Investor / Funder', id: 'OChccW' }),
+        description: intl.formatMessage({
+          defaultMessage: 'I want to find opportunities to invest / donate',
+          id: 'nhAnF3',
+        }),
+        imageSrc: '/images/account-types/investor.svg',
+        imageTitle: intl.formatMessage({
+          defaultMessage: 'Drawing of a person looking at a list of card',
```

Comment:
You can ignore this if you don't agree: 
```suggestion
          defaultMessage: 'Illustration of a person looking at a list of cards',
```